syntax = "proto3";

service NetworkManager {
    rpc GetConnectionList (GetConnectionListRequest) returns (GetConnectionListResponse) {}
    rpc GetNodeMetaData (GetNodeMetaDataRequest) returns (GetNodeMetaDataResponse) {}
}

message GetConnectionListRequest {
    string node_ip = 1;
}

message GetConnectionListResponse {
    string node_ip = 1;
}

message GetNodeMetaDataRequest {
    string node_ip = 1;
}

message GetNodeMetaDataResponse {
    string node_meta_dict = 1;
}